docs/excel/excel-add-ins-events.md

```diff
@@ -133,27 +133,27 @@ function remove() {
 > [!NOTE]
 > This feature is currently available only in public preview (beta). To use it, you must reference the beta library of the Office.js CDN: https://appsforoffice.microsoft.com/lib/beta/hosted/office.js.
 
-Events are turned on and off at the [runtime](https://docs.microsoft.com/en-us/javascript/api/excel/excel.runtime?view=office-js) level. 
+Some scenarios benefit from the performance boost given by turning off events. 
+Your app might never need to receive events, or it could ignore events while performing batch-edits of multiple entities. 
+
+Events are turned on and off at the [runtime](https://docs.microsoft.com/javascript/api/excel/excel.runtime) level. 
 The `enableEvents` property determines if events are fired and their handlers are activated. 
-Turning events off is useful when performance is critical or when editing multiple entities and want to avoid firing events until you have finished.
 
 The following code sample shows how to toggle events on and off.
 
-```typescript
-async function toggleEvents() {
-    await Excel.run(async (context) => {
-        context.runtime.load("enableEvents");
-        await context.sync();
-        const eventBoolean = !context.runtime.enableEvents
-        context.runtime.enableEvents = eventBoolean;
-        if (eventBoolean) {
-            console.log("Events are currently on.");
-        } else {
-            console.log("Events are currently off.");
-        }
-        await context.sync();
-    });
-}
+```js
```

Comment:
The language specified for this code sample is "JavaScript" -- which is what we want, since all other code samples on the page are JavaScript.

However, the code sample itself is TypeScript (e.g., uses `await`).  Suggest modifying the code sample to be pure JavaScript, so that it's consistent with other code samples on this page (and throughout our conceptual doc set).

Reply:
Doesn't JS support await/async? Our out-of-the-box yo office JavaScript file uses it.

```
async function run() {
    try {
            await Excel.run(async context => {
                /**
                 * Insert your Excel code here
                 */
                const range = context.workbook.getSelectedRange();

            // Read the range address
            range.load('address');

            // Update the fill color
            range.format.fill.color = 'yellow';

            await context.sync();
            console.log(`The range address was ${range.address}.`);
            });
        } catch(error) {
            OfficeHelpers.UI.notify(error);
            OfficeHelpers.Utilities.log(error);
        };
}
```

Reply:
@AlexJerabek @kbrandl This blog post seems to say that async/await is in ES8 (ES2017) and that it has been released, but I don't necessarily trust blog posts.

Reply:
ES7 supports `async` / `await` but I believe it requires transpile. Can you clarify this with Michael (and share your learnings with the team via email)?  i.e., what are the requirements for using `async` / `await` in vanilla JavaScript (e.g., does E7 and higher support it, but only with transpile)? This is something we should all understand and be on the same page about. Thanks!

Reply:
I rewrote the sample to avoid using async/await.
We should figure out how to reconcile the new ES7 standard with our docs. It's also something to look into that yo office uses async/await in their JavaScript, but the VS projects do not. There's a need for consistency across those project scaffolds and our docs.

Reply:
Why "the new ES7"? ES8 and ES9 have both been released and they support async/await.

Reply:
Sounds good, thanks @AlexJerabek. +1 on needing to identify the best path forward re using async/await in our docs, and ensuring consistency across both docs and tooling.

Reply:
@Rick-Kirkham Sorry, I meant ES8.